modules/ROOT/pages/getting-started.adoc

```diff
@@ -0,0 +1,127 @@
+== Fedora CoreOS - Getting Started
+
+Fedora CoreOS has no install-time configuration. All Fedora CoreOS installs or cloud images start the same, then on first boot Ignition will read the supplied config and configure the system. Ignition configs are usually supplied via the cloud’s userdata mechanism, or, in the case of bare metal, injected at install time. This guide will show you how to launch Fedora CoreOS on AWS, QEMU, and bare metal as well as how to create Ignition configs.
```

Comment:
`start the same` is awkward.

Reply:
Agreed. I want to call out
*) BM and cloud images aren't different
*) Nothing interesting happens on install

What about: "Every Fedora CoreOS instance, whether it be on a cloud or installed on bare metal, starts from the same disk image. On first boot Ignition..."

Not sure if we want to call out the _slight_ differences in the images or not.

Reply:
"Every Fedora CoreOS system begins with a generic, unconfigured disk image.  On first boot, Ignition"?